gui/src/app/pages/HomePage/LeftPanel.tsx

```diff
@@ -46,43 +46,57 @@ const LeftPanel: FunctionComponent<LeftPanelProps> = ({
     handleClose: loadProjectClose,
   } = useModalWindow();
 
-  if (collapsed) {
-    return (
-      <div className="LeftMenu" style={{ width, height }}>
-        <div className="LeftMenuContentWrapper">
-          <ExpandButton onClick={() => onSetCollapsed(false)} />
-        </div>
-      </div>
-    );
-  }
-
   return (
-    <div className="LeftMenu" style={{ width, height }}>
-      <div className="LeftMenuContentWrapper">
-        <CollapseButton onClick={() => onSetCollapsed(true)} />
-        <h3>Examples</h3>
+    <Drawer
+      open={!collapsed}
+      variant="persistent"
+      sx={{
+        width: drawerWidth,
+        flexShrink: 0,
+        "& .MuiDrawer-paper": {
+          width: drawerWidth,
+          boxSizing: "border-box",
+        },
+      }}
+    >
+      {/* For spacing purposes */}
+      <Toolbar />
 
-        {examplesStanies.map((stanie, i) => (
-          <div key={i} className="LeftMenuContentWrapper">
-            <Link replace to={`?project=${stanie.link}`}>
-              {stanie.name}
-            </Link>
-          </div>
-        ))}
-        <hr />
-        <div>
-          <button onClick={loadProjectOpen} disabled={hasUnsavedChanges}>
+      <h3>Examples</h3>
+
+      {examplesStanies.map((stanie, i) => (
```

Comment:
We may want to rename this variable now that the `Stanies` no longer exist in the project